kDrive/UI/Controller/Files/File List/InMemoryFileListViewModel.swift

```diff
@@ -40,10 +40,12 @@ class InMemoryFileListViewModel: FileListViewModel {
         }
 
         super.init(configuration: configuration, driveFileManager: driveFileManager, currentDirectory: currentDirectory)
+
+        guard let liveDirectory = try? currentDirectory.thaw() else { return }
```

Comment:
This does not fix anything. Sometimes we have a detached object and it works, sometimes we have a frozen one.
We probably should .detach all the time and it should work.